csharp/test/Apache.Arrow.Flight.TestWeb/TestFlightServer.cs

```diff
@@ -40,6 +40,15 @@ public override async Task DoAction(FlightAction request, IAsyncStreamWriter<Fli
                 case "test":
                     await responseStream.WriteAsync(new FlightResult("test data"));
                     break;
+                case "BeginTransaction":
```

Comment:
Use existing constants where available and add new ones that are standard like `Commit`

Reply:
done